src/components/global/folders/folder-info.tsx

```
@@ -1,22 +1,95 @@
+    ['rename-folder'],
+    (data: { name: string }) => renameFolders(folderId, data.name),
+    'folder-info',
+    handleError
+  )
+
+  const handleRename = () => {
+    setIsEditing(true)
+  }
+
+  const handleSave = () => {
+    if (inputRef.current && inputRef.current.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)